policy/modules/services/jabber.te

```diff
@@ -39,6 +39,7 @@ allow jabberd_domain self:tcp_socket { accept listen };
 
 manage_files_pattern(jabberd_domain, jabberd_var_lib_t, jabberd_var_lib_t)
 allow jabberd_domain jabberd_var_lib_t:dir manage_dir_perms;
+allow jabberd_domain jabberd_var_lib_t:sock_file create;
```

Comment:
Wouldn't the domain also delete the socket on shutdown? It seems like this should be `manage_sock_file_perms` instead.

Reply:
done. I had the unlink access in my toadd.te file on my jabber server. ;)